.github/workflows/ci.yml

```diff
@@ -30,6 +30,9 @@ jobs:
       contents: read
     outputs:
       all_versions: ${{ steps.version.outputs.all_versions }}
+      current: ${{ steps.version.outputs.current }}
+      oldversions: ${{ steps.version.outputs.oldversions }}
```

Comment:
Thanks, applied to https://github.com/rffontenelle/python-docs-tx-translations/pull/37 because I can't commit in merged PR (according to GitHub tooltip)